.github/workflows/containers.yml

```diff
@@ -0,0 +1,37 @@
+name: Containers
+
+on: [push, pull_request]
+
+jobs:
+  containers:
+    name: Buildah
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        gtk-version: [gtk3, gtk4]
+        linux-distro: [debian, fedora, opensuse, ubuntu]
+    env:
+      GTK_VERSION: ${{ matrix.gtk-version }}
+      LINUX_DISTRO: ${{ matrix.linux-distro }}
+      ARTIFACT_DIR: "${{ github.workspace }}/AppImages"
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Build container
+        run: |
+          mkdir -pv "${ARTIFACT_DIR}"
+          buildah bud \
```

Comment:
What's the difference to the more standard Docker command line? I haven't heard of this tool until 5 minutes ago.

Reply:
Buildah build rootless containers easily.
Docker on GitHub Actions requires [QEMU to work](https://github.com/marketplace/actions/build-and-push-docker-images#git-context), that is overkill.

Reply:
I remember the "why" now.
Buildah allows to mount volume during the build, while `docker build` does not.
The idea is to mount a volume under `/AppImage` during the container build, so all built AppImages are available in this directory.
Then, we can create artifacts from AppImages easily (because `docker cp` only works with running containers, not with images).

Reply:
Why do you need to build the AppImages during the container build, though? The typical workflow is to build a Docker image with the dependencies, then run an instance that executes a build script.

Anyway, I don't really care, let's merge this!